### PR TITLE
Update geospatial docs

### PR DESCRIPTION
**Pull request by jagill**
ST_Point signature is `double, double`, which is not clear whether that
should be `latitude, longitude` or `longitude, latitude`.  This message
clarifies that.

cc @mbasmanova

### CODE REVIEW

File: presto-docs/src/main/sphinx/functions/geospatial.rst [hunk 1]
```diff
@@ -80,7 +80,7 @@ Constructors
 
 .. function:: ST_Point(double, double) -> Point
```
Comment (mbasmanova) [inline presto-docs/src/main/sphinx/functions/geospatial.rst]: perhaps, change this signature to `ST_Point(x, y) -> Point`

File: presto-docs/src/main/sphinx/functions/geospatial.rst [hunk 2]
```diff
@@ -80,7 +80,7 @@ Constructors
...
 
-    Returns a geometry type point object with the given coordinate values.
+    Returns a geometry type point object with the given x and y.  For geospatial functions, the order is longitude then latitude.
```
Comment (mbasmanova) [inline presto-docs/src/main/sphinx/functions/geospatial.rst]: This comment applies to all ST_ functions, hence, it seems inconsistent to make it at a function level. How would you feel about changing (double, double) to (x, y) instead?

Comment (jagill) [reply presto-docs/src/main/sphinx/functions/geospatial.rst]: Do we want to maintain the type information, like `(x: double, y: double)` (or similar).  Or should that also be in the top-level description?

Comment (mbasmanova) [reply presto-docs/src/main/sphinx/functions/geospatial.rst]: It looks like we don't do this anywhere in the docs. I think (x, y) will be fine.

**Comment by jagill** [comment]
Added:
1. Formatting lines to 80 chars.
2. Standardizing format for `null`, which had three variants in that file.

**Pull request merged**